internal/restconfig/restconfig.go

```diff
@@ -72,6 +72,8 @@ type Producer struct {
 	deprecatedKubeContextsMessage *string
 	defaultNamespace              *string
 	deprecatedNamespaceFlag       *string
+	prefixedNamespace             *string
+	prefixedNamespaceFlag         bool
```

Comment:
This isn’t necessary: enabling the namespace flag automatically adds prefixed namespace flags.

Reply:
what if `namespace` flag is not enabled? i. e the default ns is not needed. Just the prefixed ns.

Reply:
If we ever need that we can add it then.